code/modules/clothing/clothing.dm

```diff
@@ -195,48 +201,71 @@ BLIND     // can't see anything
 	female_clothing_icon 			= fcopy_rsc(female_clothing_icon)
 	GLOB.female_clothing_icons[index] = female_clothing_icon
 
-/obj/item/clothing/under/verb/toggle()
-	set name = "Adjust Suit Sensors"
-	set category = "Object"
-	set src in usr
-	var/mob/M = usr
-	if (istype(M, /mob/dead/))
+/obj/item/clothing/under/proc/set_sensors(mob/user)
+	var/mob/M = user
+	if(M.stat != CONSCIOUS)
```

Comment:
use M.concious or M.isconcious its one of the 2

Reply:
Turns out m.stat works fine